client/coral-admin/src/routes/Community/containers/Community.js

```diff
@@ -20,15 +26,34 @@ class CommunityContainer extends Component {
   }
 
   render() {
-    return (
-      <Community {...this.props} />
-    );
+    return <Community {...this.props} />;
```

Comment:
Also better be explicit. Container props and Presentational props can diverge. Explicitly passing props would make this future proof and more clear.